pyproject.toml

```diff
@@ -21,21 +21,16 @@ classifiers = [
 ]
 dependencies = [
     "fire",
-    "loguru",
```

Comment:
I've used logging in the rover and spxmod stages I'm working on, but I can add it back when I make my PR. We can add the others back as needed!